.pylintrc

```
@@ -3,6 +3,7 @@
 disable = invalid-name, import-error, too-many-statements, too-many-locals, too-many-branches, too-many-arguments, eval-used, too-many-nested-blocks, too-many-public-methods, redefined-builtin, wildcard-import, function-redefined, unused-wildcard-import, duplicate-code, too-many-instance-attributes, too-few-public-methods, pointless-string-statement, no-name-in-module, no-self-use, arguments-differ, too-many-lines
 
 extension-pkg-whitelist=numpy
+max-line-length=79
```

This should be 100 othewise we will get a lot of linter errors.
```suggestion
max-line-length=100
```